src/en/resetscans/src/eu/kanade/tachiyomi/extension/en/resetscans/ResetScans.kt

```diff
@@ -1,13 +1,14 @@
 package eu.kanade.tachiyomi.extension.en.resetscans
-import eu.kanade.tachiyomi.multisrc.fuzzydoodle.FuzzyDoodle
 
-class ResetScans : FuzzyDoodle(
+import eu.kanade.tachiyomi.multisrc.madara.Madara
+
+class ResetScans : Madara(
     "Reset Scans",
-    "https://reset-scans.xyz",
+    "https://rspro.xyz",
     "en",
 ) {
```

Comment:
Missing `dateFormat`

Reply:
Added but year is not defined so is the current one by default

Reply:
The convention is to leave out the `yyyy` if it is not on the site, making it default to 1970.

I'd argue that it should be obvious that the year is unknown, rather than guessing the most recent one. Then you would run into issues with the year wrapping around multiple times if chapters span that long.

Reply:
Removed, now the year is by default 1970